m2cgen/assemblers/utils.py

```diff
@@ -19,14 +19,30 @@ def sub(l, r, to_reuse=False):
     return ast.BinNumExpr(l, r, ast.BinNumOpType.SUB, to_reuse=to_reuse)
 
 
+def lt(l, r):
+    return ast.CompExpr(l, r, ast.CompOpType.LT)
+
+
 def lte(l, r):
     return ast.CompExpr(l, r, ast.CompOpType.LTE)
 
 
+def gt(l, r):
+    return ast.CompExpr(l, r, ast.CompOpType.GT)
+
+
+def gte(l, r):
+    return ast.CompExpr(l, r, ast.CompOpType.GTE)
+
+
 def eq(l, r):
     return ast.CompExpr(l, r, ast.CompOpType.EQ)
 
 
+def ne(l, r):
```

Comment:
Is this function used? I couldn't find where

Reply:
Nope :-| 
I added all possible comparison functions for the completeness. Do you think that unused functions should be removed?

Reply:
In general I'd argue that we should introduce code only if it's being used, even when it seems like unused code contributes into the perception of completeness. This is still an unused code. I'm also guilty of doing this in the past, btw :D I suggest we introduce these functions once we have application for them.

Reply:
> I suggest we introduce these functions once we have application for them.

OK, no problem!